src/forms/inputs/checkbox-group.js

```diff
@@ -88,17 +91,45 @@ const propTypes = {
   className: PropTypes.string,
   checkboxInputProps: PropTypes.object,
   options: fieldOptionsType,
+  useDropdown: PropTypes.bool,
 }
 
 const defaultProps = {
   className: 'CheckboxGroup',
   checkboxInputProps: {},
   options: [],
+  useDropdown: false,
 }
 
-function CheckboxGroupLegend({ name, label }) {
-  if (label === false) return null
-  return <legend>{label || convertNameToLabel(name)}</legend>
+function CheckboxGroupLegend({
+  label,
+  name,
+  required,
+  requiredIndicator,
+  hint,
+}) {
+  return (
+    <legend className={classnames({ 'visually-hidden': label === false })}>
+      {label || convertNameToLabel(name)}
+      {required && requiredIndicator && (
+        <span className="required-indicator" aria-hidden="true">
+          {requiredIndicator}
+        </span>
+      )}
+      {hint && <i>{hint}</i>}
+    </legend>
+  )
+}
+
+function CheckboxOptionContainer({ children, useDropdown, value }) {
```

Comment:
Nit: Should we pluralize `Option` in `CheckboxOptionContainer` since this wraps all of the options?